### PR TITLE
docs: update kubernetes installation with multiple solution

### PR DESCRIPTION
**Pull request by jplanckeel**
## Description

We want add all solution to deploy Homer in kubernetes Dashboard

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality) 
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I've read & comply with the [contributing guidelines](https://github.com/bastienwirtz/homer/blob/main/CONTRIBUTING.md)
- [x] I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers. 
- [x] I have made corresponding changes to the documentation (README.md).
- [ ] I've checked my modifications for any breaking changes, especially in the `config.yml` file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,10 +44,19 @@
...
 </p>
 
+/**
+ * @file README.md
+ * @brief Table of Contents for the Homer project.
+ *
+ * This file contains the table of contents for the Homer project's README.md file.
+ * It provides an overview of the different sections and links to the corresponding documentation files.
+ */
+
```
Comment (bastienwirtz) [inline README.md:54]: I'm not sure if it's supposed to be displayed or not but I can see on your [repo](https://github.com/jplanckeel/homer/blob/docs/update-kubernetes-installation/README.md) that it is rendered kinda broken. 

I don't think this part is necessary, could you remove it? Thanks!

**Review by bastienwirtz — CHANGES_REQUESTED** [review]
Small issue on the README file but aside from that, it looks good !
Thank you for your contribution @jplanckeel !

File: README.md (hunk 1 above)
Comment (jplanckeel) [reply README.md]: sorry it's an involuntary addition, i have remove it ;)

**Review by bastienwirtz — APPROVED** [review]
Thanks again for improving the documentation !

**Pull request merged**